builtins/amp-video.js

```diff
@@ -45,11 +52,20 @@ export function installVideo(win) {
             'No "poster" attribute has been provided for amp-video.');
       }
 
+      videoManagerForDoc(this.win.document).register(this);
```

Comment:
Also, what concerns me about `register` is the risk conditions. We obviously don't have it with `amp-video` and not with `autoplay`, but others elements/operations might have it. For instance, "hero" videos, as determined by the video manager, would have to wait for an element to be upgraded/built which might cause some risk conditions if video manager needs to make a decision which video to play. Similarly, if we need to implement a feature "only autoplay one video at a time", this creates risk condition if two videos are in the viewport.